### ISSUE TITLE
Not accessible for screen readers

### ISSUE THREAD

**Issue by sgjudd** [SalesforceFoundation/Volunteers-for-Salesforce#324]
We have a volunteer who uses JAWS for Windows and is unable to report hours. In looking, there are issues like form labels not having corresponding identifier for the input field. Is fixing accessibility something that can be added to the queue?

**Comment by judisohn** [comment]
**lurch: attach W-020907

**Comment by judisohn** [comment]
Thanks @sgjudd This is on our backlog already, but I can see that it wasn't connected with a Github issue. I've connected it, and hopefully it will be something developers can address in the future. Thanks for reporting.
